src/core/context.py

```diff
@@ -100,7 +135,10 @@ class Status(Enum):
     """Class bundling all statuses that the charm may fall into."""
 
     WAITING_PEBBLE = MaintenanceStatus("Waiting for Pebble")
-    INVALID_CREDENTIALS = BlockedStatus("Invalid S3 credentials")
+    INVALID_S3_CREDENTIALS = BlockedStatus("Invalid S3 credentials")
```

Comment:
**thought** since we would have only one relation, maybe it is enough to have one status for "Invalid credentiaals" in general, not for specific ones. But it is not a big point, it was just to keep less statuses, but if you guys prefer to be more explicit, I'm happy either way

Reply:
For now, I'm leaving it as it is. (ie. explicit statuses for invalid s3 and azure storage credentials)